src/main/resources/export-build.qute.gradle

```diff
@@ -42,4 +42,19 @@ dependencies {
 
 application {
 	mainClass = '{fullClassName}'
+	applicationDefaultJvmArgs = [{jvmArgs}]
+{#if enablePreview eq 'true'}
+	applicationDefaultJvmArgs += ['--enable-preview']
+{/if}
 }
+
+{#if language eq 'java'}
+compileJava {
+{#if enablePreview eq 'true'}
+	options.compilerArgs += ['--enable-preview']
+{/if}
+{#if compilerArgs ne ""}
```

Comment:
Compiler args are not only for Java, they can also be for Kotlin or Groovy

Reply:
Functionality implemented and tested with the following language specific compiler options:

Groovy
* //COMPILE_OPTIONS --tolerance=5

Kotlin
* //COMPILE_OPTIONS -progressive